src/main/java/com/oracle/nosql/spring/data/repository/support/SimpleNosqlRepository.java

```diff
@@ -256,6 +256,8 @@ public String getDurability() {
     }
 
     static String convertDurability(Durability durability) {
+        Assert.notNull(durability, "durability should not be null.");
+
         if (durability.equals(Durability.COMMIT_NO_SYNC)) {
```

Comment:
Wouldn't it be better this way?
if (Durability.COMMIT_NO_SYNC.equals(durability))

Reply:
But, we still need to check for null as durability is referenced in durability.toString();